diff --git a/node_modules/pdf-lib/es/api/errors.js b/node_modules/pdf-lib/es/api/errors.js
index c9b0b01..ad9abe4 100644
--- a/node_modules/pdf-lib/es/api/errors.js
+++ b/node_modules/pdf-lib/es/api/errors.js
@@ -140,6 +140,7 @@ var CombedTextLayoutError = /** @class */ (function (_super) {
     return CombedTextLayoutError;
 }(Error));
 export { CombedTextLayoutError };
+
 var ExceededMaxLengthError = /** @class */ (function (_super) {
     __extends(ExceededMaxLengthError, _super);
     function ExceededMaxLengthError(textLength, maxLength, name) {
@@ -149,7 +150,7 @@ var ExceededMaxLengthError = /** @class */ (function (_super) {
         return _this;
     }
     return ExceededMaxLengthError;
-}(Error));
+}(ExceededMaxLengthError));
 export { ExceededMaxLengthError };
 var InvalidMaxLengthError = /** @class */ (function (_super) {
     __extends(InvalidMaxLengthError, _super);
